server/src/main/java/org/apache/iotdb/db/metadata/mtree/traverser/collector/TSEntityPathCollector.java

```diff
@@ -67,10 +65,10 @@ protected boolean processFullMatchedMNode(IMNode node, int idx, int level) {
     if (!node.isMeasurement()) {
       return false;
     }
```

Comment:
TSEntityPathCollector -> EntityPathCollector

Reply:
I renamed this class to BelongedEntityPathCollector. This will describe the task more clearly and this class do have difference with EntityPathCollector. The input pattern is to match measurement rather than entity.